src/scores/probability/brier_impl.py

```diff
@@ -61,3 +67,155 @@ def brier_score(
         check_binary(obs, "obs")
 
     return mse(fcst, obs, reduce_dims=reduce_dims, preserve_dims=preserve_dims, weights=weights)
+
+
+def ensemble_brier_score(
+    fcst: XarrayLike,  # type: ignore
+    obs: XarrayLike,  # type: ignore
+    ensemble_member_dim: str,
+    thresholds: Union[float, int, Sequence[float]],
+    *,  # Force keywords arguments to be keyword-only
+    reduce_dims: Optional[FlexibleDimensionTypes] = None,
+    preserve_dims: Optional[FlexibleDimensionTypes] = None,
+    weights: Optional[xr.DataArray] = None,
+    fair_correction: bool = True,
```

Comment:
maybe make this `apply_fair_correction_when_single_member_to_avoid_div0`. I assume people will mostly want the default, so I'd think about making this highly descriptive even though it's long. It's reasonable to disagree with me here, I'm just drawing it out for consideration.

Reply:
This argument isn't about avoiding a divide by zero error. It is to determine which equation (out of the two in the docstring) is to be used